Projects/Playground/Playground.Forms.Wpf/MainWindow.xaml

```diff
@@ -1,14 +1,13 @@
-﻿<views:MvxFormsWindowsPage
-    xmlns:views="clr-namespace:MvvmCross.Forms.Platforms.Wpf.Views;assembly=MvvmCross.Forms.Platforms.Wpf"
-    x:Class="Playground.Forms.Wpf.MainWindow"
-        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
-        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
-        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
-        xmlns:local="clr-namespace:Playground.Forms.Wpf"
-        mc:Ignorable="d"
-        Title="MainWindow" Height="350" Width="525">
+﻿<views:MvxFormsWindowsPage xmlns:views="clr-namespace:MvvmCross.Forms.Platforms.Wpf.Views;assembly=MvvmCross.Forms.Platforms.Wpf"
+                           x:Class="Playground.Forms.Wpf.MainWindow"
+                           xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+                           xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                           xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+                           xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+                           mc:Ignorable="d"
+                           Title="MainWindow"
+                           Height="350"
+                           Width="525">
     <Grid>
-        
     </Grid>
-</views:MvxFormsWindowsPage>
```

Comment:
These style changes don't provide any significant improvement, please revert.

Reply:
okey! I will change that